src/MSBuild/XMake.cs

```diff
@@ -2646,6 +2646,15 @@ private static void StartLocalNode(CommandLineSwitches commandLineSwitches)
                         OutOfProcTaskHostNode node = new OutOfProcTaskHostNode();
                         shutdownReason = node.Run(out nodeException);
                     }
+                    else if (nodeModeNumber == 3)
```

Comment:
is 3 some magic number? :)

Reply:
Pretty much. It is third mode which is introduced. Haven't found any documentation for that, where I could describe the meaning of it.

Reply:
nodeMode is ~only used by internal folks, so there isn't really documentation.